packages/react-components/react-utilities/src/hooks/useAnimationFrame.ts

```diff
@@ -17,12 +17,12 @@ const cancelAnimationFrameNoop = (handle: number) => handle;
  *
  * @returns A pair of [requestAnimationFrame, cancelAnimationFrame] that are stable between renders.
  */
-export function useAnimationFrame(): readonly [(fn: () => void, delay?: number) => number, () => void] {
+export function useAnimationFrame(): readonly [(fn: FrameRequestCallback) => number, () => void] {
```

Comment:
Can you please create a separate PR for this?

Reply:
Done.
New PR for `useAnimationFrame` types is #35466.